pyprof/nvtx/nvmarker.py

```diff
@@ -168,10 +168,11 @@ def get_trace_info(op_name):
             # Build traceMarker
             #
 
-            # Don't include any functions in this file
+            # Don't include any functions from this file (nvmarker.py)
+            # Also skip repeated back to back cases of the same file/line (recursive calls)
             #
             fnl = "{}:{}".format(frame.filename, frame.lineno)
-            if (not frame.filename.endswith("nvmarker.py")):
+            if (not frame.filename.endswith("nvmarker.py") and fnl != prev_fnl):
```

Comment:
Parentheses around the if statement are not necessary: https://docs.python.org/3/reference/expressions.html#operator-precedence

Reply:
The paren wasn't touched in this MR.